python/dgl/nn/pytorch/utils.py

```diff
@@ -555,3 +555,154 @@ def forward(self, g, labels, mask=None):
                     y[mask] = labels[mask]
 
             return y
+
+
+class LaplacianPosEnc(nn.Module):
+    r"""Laplacian Positional Encoder, as introduced in
+    `GraphGPS: General Powerful Scalable Graph Transformers
+    <https://arxiv.org/abs/2205.12454>`__
+    This module is a learned laplacian positional encoding module using Transformer or DeepSet.
+
+    Parameters
+    ----------
+    model_type : str
+        Encoder NN model type for LPE, can only be "Transformer" or "DeepSet".
+    num_layer : int
+        Number of layers in Transformer/DeepSet Encoder.
+    max_freqs : int
+        Number of eigenvectors.
```

Comment:
```suggestion
    k : int
        Number of smallest non-trivial eigenvectors.
```
Unify the notation of this parameter with `LaplacianPE` as `k` may be better. Please modify the notation in other places accordingly.